src/sdk-reference/js/6/getting-started/webpack/index.md

```diff
@@ -0,0 +1,275 @@
+---
+layout: full.html.hbs
+algolia: true
+title: Webpack
+description: Getting started with Kuzzle and Webpack
+---
+
+# Getting Started with Kuzzle and Webpack
+
+In this tutorial you will learn how to install, run and use **Kuzzle** with the **Javascript SDK** in the browser using **Webpack**.
+We will walk you through creating scripts that can **store** documents in Kuzzle and subscribe to **notifications** for each new document created.
+
+## Running Kuzzle
```

Comment:
I would remove this section and just add a note that have a Kuzzle running is a prerequisite, with a link to https://docs-v2.kuzzle.io/guide/1/getting-started/running-kuzzle/ that has the exact same text